src/experiment.rs

```diff
@@ -93,14 +93,15 @@ impl Experiment {
                     }
                 }
             }
-            AddPulls { strain_threshold } => {
-                self.add_pulls = None;
-                let new_pulls = self.fabric.measures_to_pulls(*strain_threshold);
-                self.fabric = self.frozen_fabric.take().unwrap();
-                for (alpha_index, omega_index, ideal) in new_pulls {
-                    self.fabric.create_interval(alpha_index, omega_index, Link::Pull { ideal });
-                }
-                self.start_pretensing()
+            AddPulls { .. } => {
+                unimplemented!();
+                // self.add_pulls = None;
+                // let new_pulls = self.fabric.measures_to_pulls(*strain_threshold);
+                // self.fabric = self.frozen_fabric.take().unwrap();
+                // for (alpha_index, omega_index, ideal) in new_pulls {
+                //     self.fabric.create_interval(alpha_index, omega_index, Link::Pull { ideal, material: 1 });
+                // }
+                // self.start_pretensing()
```

Comment:
What's the story here?

Reply:
Still must figure out how to do this now.  Thought I'd keep the comment for reference.